src/components/ScrollDownTag/styles.module.css

```diff
@@ -1,20 +1,25 @@
 .tagContainer {
   color: $primary-color;
   position: absolute;
-  bottom: 5px;
-  left: 50%;
-  transform: translate(-50%, -50%);
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
   animation: breathe-animation 2s ease-in-out infinite;
   transform-style: preserve-3d;
+  pointer-events: none;
+}
+
+.iconButton {
+  pointer-events: auto;
+  z-index: 1;
```

Comment:
z-index one is a little dangerous, can we do this a different way either using css absolute/relative positioning or arranging it via grid or flexbox?